README.md

```diff
@@ -269,6 +269,14 @@ profile.registerType(classToRegister, locationToRegister, 'container', 'listed')
     // querying the registry now will include the new container
     profile.typeRegistryForClass(vocab.sioc('Post'))
   })
+
+// To remove the same class from registry:
+var classToRemove = vocab.sioc('Post')
+profile.unregisterType(classToRemove, 'listed')
```

Comment:
That's good, but still, I think it makes sense to use a boolean if it's a param that only takes two possible options.